recipe/bld.bat

```diff
@@ -1,16 +1,21 @@
+@echo ON
+
 mkdir build
+if %ERRORLEVEL% NEQ 0 exit /b %ERRORLEVEL%
 cd build
+if %ERRORLEVEL% NEQ 0 exit /b %ERRORLEVEL%
 
 cmake .. %CMAKE_ARGS% ^
-         -G "%CMAKE_GENERATOR%" ^
-         -DBOOST_ROOT="%LIBRARY_PREFIX%" ^
+         -G Ninja ^
+         -DDCMAKE_VERBOSE_MAKEFILE=ON ^
          -DBUILD_NIFTY_PYTHON=ON ^
          -DWITH_HDF5=OFF ^
          -DWITH_Z5=ON ^
          -DWITH_ZLIB=ON ^
          -DWITH_BLOSC=ON ^
          -DCMAKE_BUILD_TYPE=Release ^
-         -DCMAKE_TRY_COMPILE_CONFIGURATION=Release ^
```

Comment:
i'm not sure what this "try_compile_flag" is, but my suspicion is that it is this removal that "fixed" things.

Reply:
This sets the compiler flags for the first test compilation. I ran into some issues on my local set-up without it, that's why included it here as well. It should be fine to remove it.

Reply:
ah got it.